### DISCUSSION TITLE
Is MediaStore.MediaColumns.DATA supported?

### DISCUSSION THREAD

**Discussion by lamyergeier** [gsantner/markor#1776] [Ideas] [answered]
Regarding supporting [zhanghai/MaterialFiles: Material Design file manager for Android](https://github.com/zhanghai/MaterialFiles) with Markor, the maintainer states that:

> I did provide the real path and it can be queried with the standard `MediaStore.MediaColumns.DATA` column for the content URI ([source](https://github.com/zhanghai/MaterialFiles/blob/e23299998dea2b05c314137de3ad46f1e58f9373/app/src/main/java/me/zhanghai/android/files/file/FileProvider.kt#L116)). Please ask Markor to support `MediaStore.MediaColumns.DATA`.

Is MediaStore.MediaColumns.DATA supported?

**Comment by gsantner** [comment, marked as answer]
Currently not. And improving support for other filemanagers is a big topic of it's own. There is need for people helping on that portion of code. If you are able to, feel free to help.

Otherwise, also Markor is a filemanager on it's own. So you can also select/open files from there.

And good to know MaterialFiles supplies the real path here, makes it easier to support :-).

**Reply by gsantner** [reply]
I am currently looking into this one and currently looks good,  like you can expect support for this in the next Markor update.

**Comment by gsantner** [comment]
No. And improving support for other filemanagers is a big topic of it's own, and there is need for people helping on that portion of code. If you are able to, feel free to help.

Otherwise, also Markor is a filemanager on it's own. So you can also select/open files from there.
